src/Ombi/ClientApp/app/requests/remainingrequests.module.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { CommonModule } from "@angular/common";
```

Comment:
What's this module for? It doesn't seem to be used anywhere and no components are being imported into it that we have created